src/mmw/apps/modeling/migrations/0020_remove_project_is_activity.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('modeling', '0019_project_gis_data'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
I don't think it would fail - the field wasn't required, and we don't specify it in the Rest Framework serializer. Any old releases that are ITSI embed viewers might have some unexpected behavior though (being able to create more than one project in an activity).

Happy to remove the migration/model change if you think that'd be best.